src/main/groovy/org/labkey/gradle/task/ModuleDistribution.groovy

```diff
@@ -153,15 +171,17 @@ class ModuleDistribution extends DefaultTask
         if (includeTarGZArchive)
         {
             tarArchives()
+            if (shouldBuildEmbeddedArchive())
+                embeddedTomcatTarArchive()
+
         }
         if (includeZipArchive)
         {
             zipArchives()
+            if (shouldBuildEmbeddedArchive())
+                embeddedTomcatZipArchive()
         }
-        if (includeWarArchive)
-        {
-            warArchives()
-        }
+
```

Comment:
Should we allow a distribution to have only the embedded archive?
```suggestion

        if (!includeZipArchive && !includeTarGZArchive && shouldBuildEmbeddedArchive())
        {
            embeddedTomcatTarArchive()
        }
```

Reply:
Possibly, but the choice of Tar archive here is sort of arbitrary (although I also wonder if we still need to support both tar and zip archives, but that's for a different disucssion).   The `includeEmbeddedArchive` property could be a string property where we provide 'tar.gz', 'zip' or both ('tar.gz,zip').  Perhaps that's better.